### ISSUE TITLE
PicView.EditTitleBar crashes app when no image is loaded.

### ISSUE THREAD

**Issue by Yevrag35** [Ruben2776/PicView#14]
**Describe the bug**
Launched the app and went to move the window by grabbing the title bar.  Inadvertently, I clicked the "No image loaded" message and that caused the app to crash.  The issue looks to be caused by PicView.EditTitleBar.EditTitleBar_Text:

```csharp
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at PicView.EditTitleBar.EditTitleBar_Text(Object sender, KeyboardFocusChangedEventArgs e)
   at System.Windows.Input.KeyboardFocusChangedEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
```

**To Reproduce**
1. Download the win-x64.7z file.
1. Launch the app from the .exe
2. Click the "No Image Loaded" area of the title bar when no image is loaded.

**Expected behavior**
If no image is loaded, no edit title bar behavior should be observed.

Nice app though :)

EDIT -- Should note that building the app from the source code does not produce this behavior.

**Comment by Ruben2776** [comment]
Thanks a lot for trying my app out! :)

I'll make a new release soon with it fixed.

**Comment by Ruben2776** [comment]
Fixed in release 0.9.9 https://github.com/Ruben2776/PicView/releases/tag/0.9.9
